update/views.py

```diff
@@ -13,6 +13,17 @@ def update_metadata(updated_at):
     with open(settings.METADATA_JSON, 'w') as f:
         json.dump(data, f)
 
+def update_tags(updated_at):
+    topics = []
+    data = {'updated_at' : updated_at}
+    for keys in json.load(open(settings.METADATA_JSON)):
+        for topic in keys.split('/'):
+            if topic not in (topics + ['unclassified', 'updated_at']):
+                topics.append(topic)
+    data['tags'] = list(map(lambda v: v.replace('_', ' ').lower(), topics))
```

Comment:
You may, also, consider `-` along with `_` as some inconsistencies may exist.

See `graph-algorithm`

Reply:
There were 2-3 cases where '-' was there and in some cases I thought it was necessary for the '-' to be there. So should I convert all '-' to ' ' too? @AdiChat

Reply:
Yes, convert all `-` to ` `. 👍